test/IntegrationTest.t.sol

```diff
@@ -34,7 +34,11 @@ contract IntegrationTest is Test {
     string public erc20AddressStr;
     string public counterpartyClient = "42-dummy-01";
 
-    uint256 public defaultAmount = 1000;
+    uint256 public defaultAmount = 1_000_000_100_000_000_001; // To account for a clear remainder
+    uint256 public evmConvertedAmount = 1_000_000_000_000_000_000;
+    uint256 public expectedRemainder = 100_000_000_001;
+    uint256 public expectedConvertedAmount = 1_000_000; // the uint256 representation of the uint64
+    uint256 public defaultSdkCoinAmount = 1_000_000; // sdkCoin amount
```

Comment:
I feel like these need natspec now since it is hard to understand which one is what

Reply:
I tried to document them a bit now, let me know if it good enough